src/cozmo/world.py

```diff
@@ -344,48 +344,81 @@ def recv_evt_face_disappeared(self, evt, *, face, **kw):
 
     #### Event Wrappers ####
 
-    async def wait_for_observed_light_cube(self, timeout=None):
+    def _find_visible_object(self, object_type):
+        for visible_object in self.visible_objects:
+            if (object_type is None) or isinstance(visible_object, object_type):
+                return visible_object
+        return None
+
+    def _find_visible_face(self):
+        for visible_face in self.visible_faces:
+            return visible_face
+        return None
+
+    async def wait_for_observed_light_cube(self, timeout=None, include_already_visible=True):
```

Comment:
can we call this `include_existing` so that it'd be the same if we applied it to other similar `wait_for...` style operators?